custom/workflow-helper/app-deployer/README.md

```diff
@@ -0,0 +1,75 @@
+## About Git-App-Deployer
+Git-App-Deployer has been used for the installation of applications.
+
+At first, the user is asked to give the namespace, typeName, operation, app, scope, and timeout.
+ 
+#### For Namespace
+Application namespace user has to pass:
+```-namespace=<application>```
+
+#### For Load-Test
+```-typeName=loadtest```
+
+#### For TypeName
+##### For Weak scenario check just pass:
+```-typeName=weak```
+In a weak scenario, it will create a single replica and Deployments for all.
+
+##### For Resilient scenario check just pass:
+```-typeName=resilient```
+In a resilient scenario, it will create two replicas of pods with Statefulsets for databases and Deployments for others.
+ 
+#### For Timeout
+Timeout is used for the termination of the application. The exceeding time by default is 300s.
+You may change the default time value e.g
+```-timeout=400```
+
+#### For Operation
+Operation will be Kubernetes CRUD operations on these resources.
+```-operation=apply```
+
+#### For App
+App is the name of application, which will be used in labels
+```-app=sock-shop```
+
+#### For Scope
+Scope for application
+```-scope=cluster```
+
+It creates a namespace and then installs the required application based on the given -namespace and -typeName.
+If namespace already exists, then it shows log and starts installing the application.
+```[Status]: Namespace already exists!```
+
+## Load-Test:
+The load test packages a test script in a container for Locust that simulates user traffic to the application. Please run it against the front-end service. The address and port of the frontend will be different and depend on which platform you've deployed to. See the notes for each deployment.
+It has been used parallelly with a chaos engine, which loads against the catalog front-end service.
+In the manifest, it is written as:
+```
+- name: install-application
+      container:
+        image: litmuschaos/litmus-app-deployer:latest
+        args: ["-namespace=loadtest"] 
+```
+
+## Now let’s see how the app-deployer works in the workflow:
+At first, the installation of App-Deployer(application installation) is performed.
+```
+- name: install-application
+      container:
+        image: litmuschaos/litmus-app-deployer:latest
+        args: ["-namespace=sock-shop","-typeName=resilient","-operation=apply","-timeout=400", "-app=sock-shop","-scope=cluster"] 
+```
+ ##### Note: 
+  - by default it will be resilient for weak provide type flagName as resilient(-typeName=weak)
+  - To delete application or loadtest correspondence operation need to pass ```-operation=delete```
+
+
+## Application details
+As of now Sock-shop and Potato-Head application have been added in a pre-defined workflow.
+Where weak and resilient cases are present.
+
+### In a weak scenario, 
+only one replica of the pod is present. After chaos injection, it will be down, and therefore accessibility will not be there, and eventually, it will fail due to front-end access.
```

Comment:
```suggestion
only one replica of the pod is present. After chaos injection, it will be down, and therefore accessibility will not be there, and eventually, it will fail due to lack of front-end access.
```